package.json

```diff
@@ -45,7 +45,7 @@
     "del": "^4.1.1",
     "express": "^4.17.1",
     "html-entities": "^1.2.1",
-    "http-proxy-middleware": "0.19.1",
+    "http-proxy-middleware": "0.20.0",
```

Comment:
renovate seems to do updating automatically...

Reply:
Maybe we should open issue in renovate

Reply:
I submitted an issue to Renovate as a question.